src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimizeBatched.h

```diff
@@ -207,6 +207,9 @@ class QMCFixedSampleLinearOptimizeBatched : public QMCLinearOptimizeBatched,
   // Output Hamiltonian and overlap matrices
   bool do_output_matrices_;
 
+  // Output Hamiltonian and overlap matrices in HDF format
+  bool do_output_matrices_hdf_;
```

Comment:
is do_output_matrices_ a master control of do_output_matrices_hdf_?
If two independent switches for text and hdf5 output are needed, rename the old one to be more explicit.

Reply:
They are independent.  The original switch was named `do_output_matrices_csv_`